rulesets/nixpkgs/require-merge-queue.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "require-merge-queue",
+  "target": "branch",
+  "source_type": "Repository",
+  "source": "NixOS/nixpkgs",
+  "enforcement": "active",
+  "conditions": {
+    "ref_name": {
+      "exclude": [],
+      "include": [
+        "~DEFAULT_BRANCH"
```

Comment:
Just for the record: we will _never\*_ be able to add wildcard branches to the include/exclude lists for this ruleset, as GitHub currently doesn't support requiring Merge Queues on a ruleset with wildcard patterns.

All listed branches must either be the default branch, or a non-wildcard pattern.

This means we can't add `refs/heads/release-*` and would instead have to add (e.g.) `refs/heads/release-25.05`.

_\*At least, until GitHub adds support._